### PR TITLE
Update dat-gui for v0.6.5

### PR DESCRIPTION
**Pull request by rroylance**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/dataarts/dat.gui/releases
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/dat-gui/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
-// Type definitions for dat.GUI v0.6.1
+// Type definitions for dat.GUI v0.6.5
```
Comment (aluanhaddad) [inline types/dat-gui/index.d.ts]: `@types` packages are not supposed to have a minor version and that version is not supposed to start with `v`. This should be `dat.GUI 0.6` which is why the build is failing.

**Comment by mhegazy** [comment]
@andy-ms looks like the tester bug. can you take a look.

**Comment by rroylance** [comment]
Accidentally removed my version of the repo. Closing this and reopening with newly forked version. Whoops.

**Comment by andy-ms** [comment]
Ref:  #16900

**Pull request closed without merging**